bug_report.php

```diff
@@ -302,6 +303,29 @@
 
 layout_page_header_begin();
 
+if ( ( ON === config_get_global( 'public_urls' ) ) && isset( $t_bug_data->token ) && ( !is_null( $t_bug_data->token ) ) ) {	
+	if ( isset( $_SERVER ) ) {
```

Comment:
Use config_get_global( 'path' ) instead of constructing the base path here manually.